vue3-apollo-quasar/src/components/RepoAbout/Topics.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <div v-if="topics" class="wrapper">
+    <span v-for="topic in topics" :key="topic" class="topic">
+      {{ topic }}
+    </span>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue';
+
+export default defineComponent({
+  name: 'Topics',
+  props: {
+    topics: Array,
+  },
+});
+</script>
+
+<style lang="scss" scoped>
+.wrapper {
```

Comment:
Can be replaced with `q-my-md` or `q-my-lg`. Same as other comments on using quasar classes